atproto/firehose/client.py

```diff
@@ -110,6 +113,8 @@ def _get_client(self):
         return connect(self._websocket_uri, max_size=_MAX_MESSAGE_SIZE_BYTES)
 
     def _get_async_client(self):
+        # NOTE: I've noticed that the close operation often takes the entire timeout for some reason
+        # By default this is 10 seconds, which is pretty long. Maybe shorten it?
```

Comment:
mb is it related to `.recv()` blocking?

Reply:
`async with self._get_async_client() as client` gives a `websockets.legacy.client.Connect` object, and I'm pretty sure the `__aexit__` method is responsible